sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/CrashesTest.java

```diff
@@ -594,7 +595,8 @@ public boolean matches(Object item) {
 
             @Override
             public boolean matches(Object item) {
-                return item instanceof HandledErrorLog && EXCEPTION.getMessage().equals(((HandledErrorLog) item).getException().getMessage())
+                return item instanceof HandledErrorLog && EXCEPTION.getMessage() != null
```

Comment:
How is it related to the changes? handled error path is not supposed to be impacted.

Reply:
Android Studio warns about potential NPE of `EXCEPTION.getMessage().equals(((HandledErrorLog) item).getException().getMessage())`, therefore I decided to clean up all occurrences in this file.